carta/session.py

```diff
@@ -383,6 +383,19 @@ def append_image(self, path, hdu=""):
         """
         return Image.new(self, path, hdu, True)
 
+    def open_images(self, *paths):
+        """Open multiple images.
+
+        Parameters
+        ----------
+        paths : {0}
+            The paths to the image files, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory. Separate them with comma.
+        """
+        path_list = list(paths)
+        self.open_image(path_list[0])
+        for path in path_list[1:]:
+            self.append_image(path)
+
```

Comment:
This method should return a list of image objects which reference the opened images, for consistency with `open_image` and `append_image`. So you should create an empty `images` list, and append the values returned by these methods to that list. You can then return the list at the end.